src/partials/dialog.settings.html

```diff
@@ -11,9 +11,47 @@ <h2 translate>settings.SETTINGS</h2>
         </md-toolbar>
         <md-dialog-content>
             <div class="md-dialog-content">
+                <section>
+                    <md-subheader class="md-accent"><span translate>settings.notifications.NOTIFICATIONS</span>
+                    </md-subheader>
+                    <md-list flex>
+                        <md-list-item>
+                            <md-checkbox
+                                    ng-disabled="!ctrl.notificationApiAvailable || ctrl.notificationPermission === false"
+                                    ng-model="ctrl.desktopNotifications"
+                                    ng-change="ctrl.setWantsNotifications(ctrl.desktopNotifications)"
+                                    aria-label="Show desktop notifications">
+                            </md-checkbox>
+                            <p translate>settings.notifications.SHOW_NOTIFICATIONS</p>
+                        </md-list-item>
+                        <md-list-item>
+                            <md-checkbox
+                                    ng-disabled="!ctrl.desktopNotifications"
```

Comment:
> If you want, you could prepend the "error" material icon to the messsage and make it red 

I copied your css `.status span`, `.status-yes/no/unknown i` from the troubleshooting page into the `_main.scss`. May come in handy somewhere else ;)